lib/ansible/plugins/action/net_config.py

```diff
@@ -72,10 +72,13 @@ def _get_working_path(self):
             cwd = self._task._role._role_path
         return cwd
 
-    def _write_backup(self, host, contents):
-        backup_path = self._get_working_path() + '/backup'
+    def _write_backup(self, host, contents, path=None):
+        if path is None:
+            backup_path = os.path.join(self._get_working_path(), 'backup')
+        else:
+            backup_path = os.path.abspath(path)
```

Comment:
I'm not sure this is how this should be tackled as it sets the wrong precedence.  If the module arg is a relative path it should be relative to the playbook or role root dir.   If the back path is absolute then this is okay.